src/main/resources/task-script-example.ts

```diff
@@ -0,0 +1,5 @@
+now="manel"
+echo $now; sleep 5
+echo $(date)
+echooo
```

Comment:
This line is really a part of the task script?

Reply:
Line four.

Reply:
yes, we need to test the executor behavior on badly formatted command.